packages/phone-number-privacy/signer/src/database/wrappers/domain.ts

```diff
@@ -0,0 +1,77 @@
+import { DB_TIMEOUT, ErrorMessage } from '@celo/phone-number-privacy-common'
+import Logger from 'bunyan'
+import { Counters, Histograms, Labels } from '../../common/metrics'
+import { getDatabase } from '../database'
+import { Domain, DOMAINS_COLUMNS, DOMAINS_TABLE } from '../models/domain'
+
+function domains() {
+  return getDatabase()<Domain>(DOMAINS_TABLE)
+}
+
+export async function setDomainDisabled(domain: string, logger: Logger): Promise<boolean> {
+  const disableDomainMeter = Histograms.dbOpsInstrumentation.labels('disableDomain').startTimer()
+  logger.debug({ domain }, 'Disabling domain')
+  try {
+    await domains()
+      .where(DOMAINS_COLUMNS.domain, domain)
+      .update(DOMAINS_COLUMNS.disabled, true)
+      .timeout(DB_TIMEOUT)
+    disableDomainMeter()
+    return true
+  } catch (err) {
+    Counters.databaseErrors.labels(Labels.update).inc()
+    logger.error(ErrorMessage.DATABASE_UPDATE_FAILURE)
+    logger.error(err)
+    disableDomainMeter()
+    throw err
+  }
```

Comment:
Just a simple suggestion to use the `finally` clause.
```suggestion
    return true
  } catch (err) {
    Counters.databaseErrors.labels(Labels.update).inc()
    logger.error(ErrorMessage.DATABASE_UPDATE_FAILURE)
    logger.error(err)
    throw err
  } finally {
    disableDomainMeter()
  }
```